### PR TITLE
MANATI-216: Visible links

### PR DESCRIPTION
**Pull request by EdgarBarrantes**
https://manati.atlassian.net/browse/MANATI-216

### CODE REVIEW

File: sites/all/themes/auge-theme/sass/partials/components/panels/_blogpage.scss [hunk 1]
```diff
@@ -12,17 +12,22 @@
...
-    &:active{
+    text-decoration: none;
+    &:active {
       color: $orange;
     }
   }
 }
+
+.views-field-field-post-category {
+
```
Comment (kporras07) [inline sites/all/themes/auge-theme/sass/partials/components/panels/_blogpage.scss]: Is this empty selector ok?

File: sites/all/themes/auge-theme/sass/partials/components/panels/_frontpage.scss [hunk 2]
```diff
@@ -64,6 +66,11 @@
       color: white;
     }
   }
+  &.panel-front-page-boxes-proximity {
+    a {
+      color: white;
```
Comment (kporras07) [inline sites/all/themes/auge-theme/sass/partials/components/panels/_frontpage.scss:71]: Curiosity: Why white instead of #FFF? Is it ok?

Comment (EdgarBarrantes) [reply sites/all/themes/auge-theme/sass/partials/components/panels/_frontpage.scss:71]: It's all the same so far as I know, the me from now would have prefered to use #fff but I'm just trying to make look a bit consistent with the old code.

**Comment by kporras07** [comment]
:shipit:

**Pull request merged**